specs/animations/master/Overview.html

```diff
@@ -3428,7 +3428,7 @@ <h3 id="InterfaceSVGAnimationElement">Interface SVGAnimationElement</h3>
     <dl class="attributes">
       <dt id="__svg__SVGAnimationElement__targetElement" class="attribute first-child"><b>targetElement</b><span class="idl-type-parenthetical"> (readonly <a>SVGElement</a>?)</span></dt>
       <dd class="attribute">
-        <div>The element which is being animated. If no target element is being animated (for example, because the <a>'animate/href'</a> specifies an unknown element) the value returned is null.</div>
```

Comment:
The "unknown" in this case refers to an unknown `id`, not an unknown tag name.  Maybe to make it clear, the parentheses could be

>  (for example, because the URL in <a>'animate/href'</a> does not match a valid element)

Reply:
Added